{"timestamp":"2025-10-11T14:13:42.236943Z","level":"info","event":"DAG bundles loaded: dags-folder, example_dags","logger":"airflow.dag_processing.bundles.manager.DagBundlesManager","filename":"manager.py","lineno":179}
{"timestamp":"2025-10-11T14:13:42.237312Z","level":"info","event":"Filling up the DagBag from /opt/airflow/dags/ETL_toll_data.py","logger":"airflow.models.dagbag.DagBag","filename":"dagbag.py","lineno":593}
{"timestamp":"2025-10-11T14:13:42.242313Z","level":"error","event":"Failed to resolve template field 'bash_command'","logger":"airflow.sdk.definitions._internal.templater","filename":"templater.py","lineno":100,"error_detail":[{"exc_type":"TemplateNotFound","exc_value":"'./data/extract.sh' not found in search path: '/opt/airflow/dags'","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/definitions/_internal/templater.py","lineno":98,"name":"resolve_template_files"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/jinja2/loaders.py","lineno":209,"name":"get_source"}],"is_group":false,"exceptions":[]}]}
{"timestamp":"2025-10-11T14:13:42.244980Z","level":"error","event":"Exception rendering Jinja template for task 'extract', field 'bash_command'. Template: './data/extract.sh'","logger":"airflow.sdk.definitions._internal.abstractoperator","filename":"abstractoperator.py","lineno":308,"error_detail":[{"exc_type":"TemplateNotFound","exc_value":"'./data/extract.sh' not found in search path: '/opt/airflow/dags'","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/definitions/_internal/abstractoperator.py","lineno":305,"name":"_do_render_template_fields"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/definitions/_internal/templater.py","lineno":174,"name":"render_template"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/jinja2/environment.py","lineno":1016,"name":"get_template"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/jinja2/environment.py","lineno":975,"name":"_load_template"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/jinja2/loaders.py","lineno":126,"name":"load"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/jinja2/loaders.py","lineno":209,"name":"get_source"}],"is_group":false,"exceptions":[]}]}
{"timestamp":"2025-10-11T14:13:42.245195Z","level":"error","event":"Task failed with exception","logger":"task","filename":"task_runner.py","lineno":994,"error_detail":[{"exc_type":"TemplateNotFound","exc_value":"'./data/extract.sh' not found in search path: '/opt/airflow/dags'","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":914,"name":"run"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":804,"name":"_prepare"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":284,"name":"render_templates"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/bases/operator.py","lineno":1569,"name":"render_template_fields"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/definitions/_internal/abstractoperator.py","lineno":305,"name":"_do_render_template_fields"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/definitions/_internal/templater.py","lineno":174,"name":"render_template"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/jinja2/environment.py","lineno":1016,"name":"get_template"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/jinja2/environment.py","lineno":975,"name":"_load_template"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/jinja2/loaders.py","lineno":126,"name":"load"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/jinja2/loaders.py","lineno":209,"name":"get_source"}],"is_group":false,"exceptions":[]}]}
{"timestamp":"2025-10-11T14:13:42.257026Z","level":"info","event":"Task instance in failure state","logger":"task.stdout"}
{"timestamp":"2025-10-11T14:13:42.257327Z","level":"info","event":"Task start","logger":"task.stdout"}
{"timestamp":"2025-10-11T14:13:42.257405Z","level":"info","event":"Task:<Task(BashOperator): extract>","logger":"task.stdout"}
{"timestamp":"2025-10-11T14:13:42.257448Z","level":"info","event":"Failure caused by './data/extract.sh' not found in search path: '/opt/airflow/dags'","logger":"task.stdout"}
{"timestamp":"2025-10-11T14:13:42.300899Z","level":"warning","event":"Using Connection.get_connection_from_secrets from `airflow.models` is deprecated.Please use `get` on Connection from sdk(`airflow.sdk.Connection`) instead","category":"DeprecationWarning","filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/models/connection.py","lineno":486,"logger":"py.warnings"}
{"timestamp":"2025-10-11T14:13:42.305842Z","level":"info","event":"Email alerting: attempt 1","logger":"airflow.utils.email","filename":"email.py","lineno":261}
{"timestamp":"2025-10-11T14:13:42.312480Z","level":"info","event":"Email alerting: attempt 1","logger":"airflow.utils.email","filename":"email.py","lineno":261}
{"timestamp":"2025-10-11T14:13:42.312850Z","level":"error","event":"Top level error","logger":"task","filename":"task_runner.py","lineno":1457,"error_detail":[{"exc_type":"ConnectionRefusedError","exc_value":"[Errno 111] Connection refused","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":1452,"name":"main"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":1419,"name":"finalize"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":1259,"name":"_send_task_error_email"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/email.py","lineno":79,"name":"send_email"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/email.py","lineno":154,"name":"send_email_smtp"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/email.py","lineno":263,"name":"send_mime_email"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/email.py","lineno":306,"name":"_get_smtp_connection"},{"filename":"/usr/python/lib/python3.12/smtplib.py","lineno":255,"name":"__init__"},{"filename":"/usr/python/lib/python3.12/smtplib.py","lineno":341,"name":"connect"},{"filename":"/usr/python/lib/python3.12/smtplib.py","lineno":312,"name":"_get_socket"},{"filename":"/usr/python/lib/python3.12/socket.py","lineno":865,"name":"create_connection"},{"filename":"/usr/python/lib/python3.12/socket.py","lineno":850,"name":"create_connection"}],"is_group":false,"exceptions":[]},{"exc_type":"ConnectionRefusedError","exc_value":"[Errno 111] Connection refused","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":1257,"name":"_send_task_error_email"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/email.py","lineno":79,"name":"send_email"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/email.py","lineno":154,"name":"send_email_smtp"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/email.py","lineno":263,"name":"send_mime_email"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/email.py","lineno":306,"name":"_get_smtp_connection"},{"filename":"/usr/python/lib/python3.12/smtplib.py","lineno":255,"name":"__init__"},{"filename":"/usr/python/lib/python3.12/smtplib.py","lineno":341,"name":"connect"},{"filename":"/usr/python/lib/python3.12/smtplib.py","lineno":312,"name":"_get_socket"},{"filename":"/usr/python/lib/python3.12/socket.py","lineno":865,"name":"create_connection"},{"filename":"/usr/python/lib/python3.12/socket.py","lineno":850,"name":"create_connection"}],"is_group":false,"exceptions":[]}]}
{"timestamp":"2025-10-11T14:13:42.323712Z","level":"warning","event":"Process exited abnormally","exit_code":1,"logger":"task"}
